Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift

```diff
@@ -55,10 +63,12 @@ class ___VARIABLE_sceneName___Interactor: ___VARIABLE_sceneName___BusinessLogic,
         }
 
         worker?.perform___VARIABLE_sceneName___(completion: {
-            [weak self] (isSuccessful, error) in
+            [weak self, request] isSuccessful, error in
```

Comment:
may I know why request here is declared weak?

Reply:
Only `self` is weak, `request` is not. We simply capture the `request` object so that it will still be available after the common long running process is completed.

Reply:
Ic... this give me idea on the upload image/pdf problem that we have last time... thanks...